### PR TITLE
Update form_validation_lang.php

### PR DESCRIPTION
**Pull request by pedrohills**
Atualizado algumas frases incorretas...
Fiz uma correção geral no arquivo..

### CODE REVIEW

File: language/portuguese-brazilian/form_validation_lang.php [hunk 1]
```diff
@@ -9,32 +9,32 @@
...
+$lang['form_validation_is_natural'] = 'O campo {field} deve conter um número natural.';
+$lang['form_validation_is_natural_no_zero'] = 'O campo {field} deve conter um número natural diferente de zero.';
 $lang['form_validation_decimal'] = 'O campo {field} deve conter um número decimal.';
 $lang['form_validation_less_than'] = 'O campo {field} deve conter um número menor que {param}';
 $lang['form_validation_less_than_equal_to'] = 'O campo {field} deve conter um número menor ou igual que {param}.';
 $lang['form_validation_greater_than'] = 'O campo {field} deve conter um número maior que {param}.';
 $lang['form_validation_greater_than_equal_to'] = 'O campo {field} deve conter um número maior ou igual que {param}.';
 $lang['form_validation_error_message_not_set'] = 'Não existe uma mensagem de erro para o campo com o nome {field}.';
-$lang['form_validation_in_list'] = 'The {field} field must be one of: {param}.'; // to translate
+$lang['form_validation_in_list'] = 'O campo {field} deve ser um de: {param}.'; // translated
```
Comment (Razican) [inline language/portuguese-brazilian/form_validation_lang.php]: I think this comment should be deleted.

**Comment by pedrohills** [comment]
No problem. :)

File: language/portuguese-brazilian/form_validation_lang.php (hunk 1 above)
Comment (pedrohills) [inline language/portuguese-brazilian/form_validation_lang.php]: No problem. =)

**Comment by jim-parry** [comment]
I don't understand the original PR message (not English), and am waiting for @mateusfbi to comment on or approve this PR.

**Comment by pedrohills** [comment]
I just fix some grammatical errors ...

**Comment by mateusfbi** [comment]
okay

**Comment by jim-parry** [comment]
okay ... i will take your "okay" as agreeing to the change, and not as "okay, i will comment".

**Pull request merged**